packages/docusaurus-utils/src/markdownUtils.ts

```diff
@@ -55,16 +55,18 @@ export function createExcerpt(fileString: string): string | undefined {
   const fileLines = fileString
     .trimStart()
     // Remove Markdown alternate title
-    .replace(/^[^\n]*\n[=]+/g, '')
-    .split('\n');
+    .replace(/^[^\r\n]*\r?\n[=]+/g, '')
+    .split(/\r?\n/);
```

Comment:
This is the fix. So far only CR was handled, not CRLF line endings.

Reply:
I assume you mean "only LF was handled" 😄

Reply:
Yes, sorry 😉